lib/api/controllers/adminController.js

```diff
@@ -218,7 +217,7 @@ class AdminController extends BaseController {
     const action = request.input.action;
 
     if (_locks[action]) {
-      throw new PreconditionError(message);
+      this.throw('precondition', message);
```

Comment:
Wrong condition name: should be something like `action_locked` or something.
As it is, reading the error configuration file makes it impossible to understand what this error does.